docs/guides/qiskit-code-assistant-local.mdx

```diff
@@ -24,18 +24,80 @@ The Open Source Qiskit Code Assistant models are available in <DefinitionTooltip
 
 Follow these steps to download any Qiskit Code Assistant-related model from the Hugging Face website:
 
-1. Navigate to the desired Qiskit model page in
-1. Go to the Files and Versions tab and download the safetensors or GGUF model files.
-
-Publicly available open source models:
-1. [qiskit/granite-3.3-8b-qiskit](https://huggingface.co/Qiskit/granite-3.3-8b-qiskit). Released June 2025.
-1. [qiskit/granite-3.3-8b-qiskit-GGUF](https://huggingface.co/Qiskit/granite-3.3-8b-qiskit-GGUF). Released June 2025. Recommended for use in personal environments/laptops
-1. [qiskit/granite-3.2-8b-qiskit](https://huggingface.co/Qiskit/granite-3.2-8b-qiskit). Released June 2025.
-1. [qiskit/granite-3.2-8b-qiskit-GGUF](https://huggingface.co/Qiskit/granite-3.2-8b-qiskit-GGUF). Released June 2025. Recommended for use in personal environments/laptops
-1. [qiskit/qwen2.5-coder-14b-qiskit](https://huggingface.co/Qiskit/qwen2.5-coder-14b-qiskit). Released June 2025.
-1. [qiskit/qwen2.5-coder-14b-qiskit-GGUF](https://huggingface.co/Qiskit/qwen2.5-coder-14b-qiskit-GGUF). Released June 2025. Recommended for use in personal environments/laptops
-1. [qiskit/granite-8b-qiskit-rc-0.10](https://huggingface.co/Qiskit/granite-8b-qiskit-rc-0.10). Released February 2025 (deprecated).
-1. [qiskit/granite-8b-qiskit](https://huggingface.co/Qiskit/granite-8b-qiskit). Released November 2024 (deprecated).
+1. Navigate to the desired Qiskit model page on Hugging Face
+2. Go to the **Files and Versions** tab and download the safetensors or GGUF model files
+
+### Available Models
+
+#### Current Models
+
+These are the latest recommended models for use with Qiskit Code Assistant:
+
+1. [Qiskit/mistral-small-3.2-24b-qiskit](https://huggingface.co/Qiskit/mistral-small-3.2-24b-qiskit) - Released October 2025
+2. [qiskit/granite-3.3-8b-qiskit](https://huggingface.co/Qiskit/granite-3.3-8b-qiskit) - Released June 2025
+3. [qiskit/granite-3.2-8b-qiskit](https://huggingface.co/Qiskit/granite-3.2-8b-qiskit) - Released June 2025
+4. [qiskit/qwen2.5-coder-14b-qiskit](https://huggingface.co/Qiskit/qwen2.5-coder-14b-qiskit) - Released June 2025
+
+#### GGUF Models (Recommended for Personal Environments/Laptops)
+
+GGUF format models are optimized for local use and require less computational resources:
+
+1. [qiskit/granite-3.3-8b-qiskit-GGUF](https://huggingface.co/Qiskit/granite-3.3-8b-qiskit-GGUF) - Released June 2025
+2. [qiskit/granite-3.2-8b-qiskit-GGUF](https://huggingface.co/Qiskit/granite-3.2-8b-qiskit-GGUF) - Released June 2025
+3. [qiskit/qwen2.5-coder-14b-qiskit-GGUF](https://huggingface.co/Qiskit/qwen2.5-coder-14b-qiskit-GGUF) - Released June 2025
```

Comment:
Since the Qwen models are more performant than the granite ones, could we reorder the lists? Sorting by date and by quality when the release date is the same